etc/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <dev>
+            <js>
+                <minify_exclude>
+                    /CloudFlare_Plugin/js/compiled.js
+                </minify_exclude>
```

Comment:
This should probably be a single line

Reply:
`Spaces must be used to indent lines; tabs are not allowed` I guess this is also a problem

Reply:
It being on a line by itself is fine, reviewing the core code \Magento\Framework\View\Asset\Minification::getExcludes.

I am curious as to whether or not this will work at all, though - since I'm fairly sure this is just detailing a default configuration value?

Might need an install script to update it, as opposed to setting a default.

Reply:
I know the previous version did not work. I haven't checked whether 3316888 works or not. 
I followed the steps provided in this link https://github.com/cloudflare/Cloudflare-Magento/issues/56#issuecomment-325154816. 

> Might need an install script to update it, as opposed to setting a default.


Can you elaborate what you meant here?